polka-index/RFC.md

```diff
@@ -0,0 +1,122 @@
+# RFC: Polka-index
```

Comment:
Who's running those nodes? I don't see any incentive to run them.
What about running indexing as part of the storage provider node - each storage provider is also an indexer?